src/consumerIncentives/ConsumerIncentivesHomeScreen.test.tsx

```diff
@@ -18,6 +19,20 @@ import { RootState } from 'src/redux/reducers'
 import { StoredTokenBalance } from 'src/tokens/slice'
 import { createMockStore } from 'test/utils'
 import { mockCeurAddress, mockCusdAddress } from 'test/values'
+
+jest.mock('react-i18next', () => ({
+  ...(jest.requireActual('../../__mocks__/react-i18next') as any),
+  Trans: ({ i18nKey, tOptions, children }: TransProps) => {
```

Comment:
i want to make this change in the __mocks__ file but will do that in a separate PR. the way we've mocked `Trans` is just to render all the child nodes, which is not helpful if you want to validate the key itself (which, i think is the more useful thing to test for)